pinax/blog/hooks.py

```diff
@@ -0,0 +1,16 @@
+from pinax.blog.conf import settings
+
+
+class PinaxBlogDefaultHookSet(object):
+
+    def get_scoped_object(self, **kwargs):
```

Comment:
Another misleading name. The returned object from this method is the scoper and not the scoped object. The scoped object is the `Post` instance.